src/env/list_delete.c

```diff
@@ -0,0 +1,29 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*   tosh - 21 Shell                                      ::::::::            */
+/*                                                      :+:    :+:            */
+/*   By: aholster <aholster@student.codam.nl>          +:+                    */
+/*       ivan-tey <ivan-tey@student.codam.nl>         +#+                     */
+/*       nloomans <nloomans@student.codam.nl>        +#+                      */
+/*                                                 #+#    #+#                 */
+/*   License: GPLv3                                ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "private.h"
+
+void					env_list_delete(t_env *env)
```

Comment:
since env itself is not a malloc'd object, the delete name is a misnomer, also, missing ptr const.

Reply:
duplicate of previous comment, resolving